specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -0,0 +1,973 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace AzureAppConfiguration;
+
+// TODO: check if service supports any of the following headers:
+alias standardOps = ResourceOperations<
+  NoClientRequestId & NoRepeatableRequests & NoConditionalRequests,
```

Comment:
I configured this so we dont add any of these headers since they arent specified in your swagger, but does your service actually support any of these headers?

Reply:
The closest doc I found about these keywords was this: https://azure.github.io/typespec-azure/docs/getstarted/azure-core/step09
Not sure what it means to support these headers, but we do support conditional requests (If-Match and If-None-Match headers).

Reply:
Here is what you'll get if we enable the client request Id header:

```
"x-ms-client-request-id": {
                "type": "string",
                "format": "uuid",
                "description": "An opaque, globally-unique, client-generated string identifier for the request."
              }
```

Here is what you'll get if we add supports conditional requests:

```
"Azure.Core.ConditionalRequestHeaders.ifMatch": {
      "name": "If-Match",
      "in": "header",
      "description": "The request should only proceed if an entity matches this string.",
      "required": false,
      "type": "string",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "ifMatch"
    },
    "Azure.Core.ConditionalRequestHeaders.ifModifiedSince": {
      "name": "If-Modified-Since",
      "in": "header",
      "description": "The request should only proceed if the entity was modified after this time.",
      "required": false,
      "type": "string",
      "format": "date-time",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "ifModifiedSince"
    },
    "Azure.Core.ConditionalRequestHeaders.ifNoneMatch": {
      "name": "If-None-Match",
      "in": "header",
      "description": "The request should only proceed if no entity matches this string.",
      "required": false,
      "type": "string",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "ifNoneMatch"
    },
    "Azure.Core.ConditionalRequestHeaders.ifUnmodifiedSince": {
      "name": "If-Unmodified-Since",
      "in": "header",
      "description": "The request should only proceed if the entity was not modified after this time.",
      "required": false,
      "type": "string",
      "format": "date-time",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "ifUnmodifiedSince"
    },
```

Does your service support all of these headers?

Reply:
We support x-ms-client-request-id, If-Match, If-None-Match headers.

Can you elaborate on NoRepeatableRequests headers?

Reply:
Adding client request id for all of your ops. The repeatability headers are: 
``` 
"Azure.Core.RepeatabilityRequestHeaders.repeatabilityFirstSent": {
      "name": "Repeatability-First-Sent",
      "in": "header",
      "description": "Specifies the date and time at which the request was first created.",
      "required": false,
      "type": "string",
      "format": "date-time",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "repeatabilityFirstSent"
    },
    "Azure.Core.RepeatabilityRequestHeaders.repeatabilityRequestId": {
      "name": "Repeatability-Request-ID",
      "in": "header",
      "description": "An opaque, globally-unique, client-generated string identifier for the request.",
      "required": false,
      "type": "string",
      "x-ms-parameter-location": "method",
      "x-ms-client-name": "repeatabilityRequestId"
    }
```

Reply:
Thanks! We dont support Repeatability headers

Reply:
sounds good! Thanks for the info